crates/loam-cli/src/commands/build/clients.rs

```diff
@@ -108,6 +126,83 @@ impl Args {
         Ok(())
     }
 
+
+    fn get_network_args(network: &Network) -> cli::network::Args {
+        cli::network::Args {
+            rpc_url: network.rpc_url.clone(),
+            network_passphrase: network.network_passphrase.clone(),
+            network: network.name.clone(),
+        }
+    }
+
+    fn get_config_locator() -> cli::config::locator::Args {
+        cli::config::locator::Args {
+            global: false,
+            config_dir: None,
+        }
+    }
+
+    fn get_config_dir(network: &Network) -> cli::config::Args {
+        let account = std::env::var("STELLAR_ACCOUNT")
+            .expect("No STELLAR_ACCOUNT environment variable set");
+        cli::config::Args {
+            network: Self::get_network_args(network),
+            locator: Self::get_config_locator(),
+            source_account: account,
+            hd_path: Some(0),
+        }
+    }
+     
+    fn get_contract_alias(&self, name: &str, network: &Network) -> Result<Option<String>, cli::config::alias::Error> {
+        let config_dir = Self::get_config_dir(network);
+        let network_passphrase = std::env::var("STELLAR_NETWORK_PASSPHRASE")
+            .expect("No STELLAR_NETWORK_PASSPHRASE environment variable set");
+        config_dir.get_contract_id(name, &network_passphrase)
+    }
+
+    async fn contract_hash_matches(&self, contract_id: &str, hash: &str, network: &Network) -> Result<bool, Error> {
+        let hash_vec = cli::contract::fetch::Cmd {
+            contract_id: contract_id.to_string(),
+            out_file: None, 
+            locator: Self::get_config_locator(),
+            network: Self::get_network_args(network),
+        }
+            .run_against_rpc_server(None, None)
+            .await?;
+        let ctrct_hash = contract_hash(&hash_vec)?;
+        Ok(hex::encode(ctrct_hash) == hash)
+    }
+
+    fn save_contract_alias(&self, name: &str, contract_id: &str, network: &Network) -> Result<(), cli::config::alias::Error> {
+        let config_dir = Self::get_config_dir(network);
+        config_dir.save_contract_id(contract_id, name)
+    }
+
+    fn write_contract_template(&self, workspace_root: &std::path::Path, name: &str, contract_id: &str) -> Result<(), Error> {
+        let allow_http = if self.loam_env() == "development" {
+            "\n  allowHttp: true,"
+        } else {
+            ""
+        };
+        let network = std::env::var("STELLAR_NETWORK_PASSPHRASE")
+            .expect("No STELLAR_NETWORK_PASSPHRASE environment variable set");
+        let template = format!(
+            r#"import * as Client from '{name}';
+    import {{ rpcUrl }} from './util';
+    
+    export default new Client.Client({{
+      networkPassphrase: '{network}',
+      contractId: '{contract_id}',
+      rpcUrl,{allow_http}
+      publicKey: undefined,
+    }});
+    "#
```

Comment:
We want this to look right in a TS file

```suggestion
import {{ rpcUrl }} from './util';
    
export default new Client.Client({{
  networkPassphrase: '{network}',
  contractId: '{contract_id}',
  rpcUrl,{allow_http}
  publicKey: undefined,
}});
"#
```

Reply:
Ah, good catch, thanks!